bijection-finagle-mysql/src/test/scala/com/twitter/bijection/finagle_mysql/MySqlConversionLaws.scala

```diff
@@ -86,4 +86,17 @@ class MySqlConversionLaws extends PropSpec with PropertyChecks with MustMatchers
   property("Null") {
     isInjection[NullValue.type, Option[String]]
   }
+  property("Timestamp") {
+    // Custom equivalence typeclass for RawValue
+    implicit val valueEquiv = new scala.math.Equiv[Value] {
+      override def equiv(a: Value, b: Value) = (a, b) match {
+        case (RawValue(Type.Timestamp, Charset.Binary, true, bytes1),
+          RawValue(Type.Timestamp, Charset.Binary, true, bytes2)) =>
+          bytes1.toList == bytes2.toList
```

Comment:
This is here to compare two `RawValue`s generated from `Timestamp`s. Because they contain byte arrays, just `==`ing them does not work as expected.